adminservice/src/main/resources/templates/dashboard.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
```

Comment:
No idea. Eclipse generated it for me and I didn't see it. I've removed the meta tag.